.devcontainer.json

```diff
@@ -1,13 +1,20 @@
 {
   "name": "Home Assistant Frontend",
-  "build": {
-    "dockerfile": "Dockerfile",
-    "context": ".."
-  },
+  "image": "mcr.microsoft.com/devcontainers/python:0-3.10",
   "appPort": "8124:8123",
```

Comment:
Consider revising the ports as well. Using this variable is no longer recommended.

Reply:
Unrelated, and there is another PR open that do that.